src/app/services/dia-backend/auth/dia-backend-auth.service.ts

```diff
@@ -30,7 +30,21 @@ export class DiaBackendAuthService {
       key: 'numbersStoragePublisher_authToken',
     });
     if (oldToken.value) {
-      this.setToken(oldToken.value);
+      this.setToken(oldToken.value.split(' ')[1]);
```

Comment:
The token will be set to `undefined` if the original token doesn't contain a space and prevents redirection to the login page as well as communication with dia-backend. Though it seems unlikely to happen, we should either make sure all previously released version has an expected token value format, or make the login check fail for `undefined` token value (if the token is malformed it's not gonna work anyway)

https://github.com/numbersprotocol/capture-lite/blob/b156980f4a0ac3a5355f82ad362d2522d03e25b9/src/app/services/dia-backend/auth/dia-backend-auth.service.ts#L138